website/pages/docs/form/editable.mdx

```diff
@@ -65,12 +65,12 @@ function CustomControlsExample() {
 
     return isEditing ? (
       <ButtonGroup justifyContent="center" size="sm">
-        <IconButton icon={<CheckIcon />} {...getSubmitButtonProps()} />
-        <IconButton icon={<CloseIcon />} {...getCancelButtonProps()} />
+        <IconButton icon={<CheckIcon />} {...getSubmitButtonProps()} aria-label="Submit" />
+        <IconButton icon={<CloseIcon />} {...getCancelButtonProps()} aria-label="Cancel" />
```

Comment:
The `getSubmitButtonProps()`, `getCancelButtonProps()` and `getEditButtonProps()` already include the corresponding `aria-label` props.

Reply:
Yep, I know that but vscode was giving me an error saying `aria-label` is a required prop even after it was getting passed by the mentioned functions